### ISSUE TITLE
spectators join game after map change

### ISSUE THREAD

**Issue by derselbst** [chuck-norris-network/teeworlds#16]
...automatically. is that intended??

**Comment by black-roland** [comment]
Yeah, it is not cool, that's why we use `sv_tournament_mode 1` on a DM servers with map rotation. A server state is zero out after map change (like after `reload` command), including spectators state, teams, scores, etc. I don't think what we can easily fix that 😕
